tests/shell/tests.py

```diff
@@ -0,0 +1,15 @@
+from django import __version__
+from django.core.management import call_command
+from django.test import SimpleTestCase
+from django.test.utils import patch_logger
+
+
+class ShellCommandTestCase(SimpleTestCase):
+
+    def test_command_option(self):
+        with patch_logger('test', 'info') as logger:
```

Comment:
Is the complexity of testing with a logger needed? I think a print statement and passing `stdout=StringIO()` to `call_command()` in order to check the value would be sufficient.